owncloudApp/src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -249,6 +247,12 @@ private int getThumbnailDimension() {
             return Math.round(r.getDimension(R.dimen.file_icon_size_grid));
         }
 
+        private String getPreviewUrl(OCFile ocFile, Account account) {
+            return mClient.getBaseUri() +
+                    "/remote.php/dav/files/" + account.name.split("@")[0] + Uri.encode(ocFile.getRemotePath(), "/")
```

Comment:
Maybe this string can be put into a constant.

Reply:
Maybe this whole line can be written as interpolatable string.

Reply:
I don't like it so much neither, but we can not use a constant. I could only use "/remote.php/dav/files/" as constant. But no the whole url since it is dynamic depending ocFile and account. About interpolatable string, well, i would like to, but this is java, not kotlin.

Reply:
I updated my solution. Please have a look.